documentation/publishing.md

```diff
@@ -1,8 +1,9 @@
 # Publishing
 
-We use Lerna to create changelogs and releases for all our packages and to publish them to npm.
+We use a Release Please GitHub action to create changelog and release PRs for all our packages.
+When the release PR is merged, that same action publishes the new release to npm, GitHub releases and deploys it to our main Storybook instance.
```

Comment:
```suggestion
When the release PR is merged, that same action publishes the new release to npm, creates a release on GitHub, and deploys it to our Storybook environment.
```

Reply:
Hm, we have several Storybook environments though. The reason I wrote 'main' is to clarify that it goes to the main one